cloud-pubsub/main.py

```diff
@@ -6,13 +6,23 @@
 """
 
 import datetime
+import time
 
 from google.cloud import pubsub
 
 PUBSUB_TOPIC = 'eco'
 PUBSUB_SUBSCRIPTION = 'echo-read'
 
 
+def process(message):
+    """Process received message"""
+    print("[{0}] Processing: {1}".format(datetime.datetime.now(),
```

Comment:
please use `message.message_id` in this method. text can be long or non-unique.

Reply:
done